app/src/main/java/org/jellyfin/mobile/player/PlayerFragment.kt

```diff
@@ -2,57 +2,62 @@ package org.jellyfin.mobile.player
 
 import android.annotation.SuppressLint
 import android.app.PictureInPictureParams
-import android.content.Context
-import android.content.Intent
 import android.content.pm.ActivityInfo
 import android.content.res.Configuration
+import android.graphics.Color
 import android.media.AudioManager
 import android.os.Build
 import android.os.Bundle
 import android.provider.Settings.System
 import android.view.*
 import android.widget.*
-import androidx.activity.viewModels
-import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.getSystemService
+import androidx.core.content.withStyledAttributes
 import androidx.core.view.ViewCompat
 import androidx.core.view.isVisible
 import androidx.core.view.postDelayed
 import androidx.core.view.updatePadding
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
 import com.google.android.exoplayer2.Player
 import com.google.android.exoplayer2.ui.AspectRatioFrameLayout
 import com.google.android.exoplayer2.ui.PlayerView
 import org.jellyfin.mobile.AppPreferences
 import org.jellyfin.mobile.R
-import org.jellyfin.mobile.databinding.ActivityPlayerBinding
 import org.jellyfin.mobile.databinding.ExoPlayerControlViewBinding
+import org.jellyfin.mobile.databinding.FragmentPlayerBinding
 import org.jellyfin.mobile.utils.*
 import org.jellyfin.mobile.utils.Constants.DEFAULT_CENTER_OVERLAY_TIMEOUT_MS
 import org.jellyfin.mobile.utils.Constants.DEFAULT_CONTROLS_TIMEOUT_MS
 import org.jellyfin.mobile.utils.Constants.DEFAULT_SEEK_TIME_MS
 import org.koin.android.ext.android.inject
+import timber.log.Timber
 import kotlin.math.abs
 
-class PlayerActivity : AppCompatActivity() {
+class PlayerFragment : Fragment() {
 
     private val appPreferences: AppPreferences by inject()
     private val viewModel: PlayerViewModel by viewModels()
-    private lateinit var playerBinding: ActivityPlayerBinding
+    private var _playerBinding: FragmentPlayerBinding? = null
+    private val playerBinding: FragmentPlayerBinding get() = _playerBinding!!
```

Comment:
Why not a `lateinit var`?

Reply:
Because you have to null it out when the view is destroyed. Also, that's the patter that the Android documentation recommends for view binding in fragments.

Reply:
Not the biggest fan but I remember them recommending something similar for LiveData/MutableLiveData.